server/server.go

```diff
@@ -307,7 +305,7 @@ func (s *Server) GetConfig(instID flux.InstanceID, includeSecrets bool) (flux.In
 
 	config := fullConfig.Settings
 	if !includeSecrets {
-		removeSecrets(&config)
+		config = config.HideSecrets()
```

Comment:
It would be nice it no secrets was the default, and you had to "ExposeSecrets()"